app/js/main.js

```diff
@@ -74,18 +74,28 @@ var width  = 640,
 
     // Notification bar
     notifyBar    = document.querySelector(".notification"),
-    notifyBarMsg = document.querySelector(".notification #msg");
+    notifyBarMsg = document.querySelector(".notification #msg"),
+    
+    // Confirm messages
+    confirmContainer = document.querySelector("#confirm-container"),
+    confirmText      = document.querySelector("#confirm-text"),
+    confirmOK        = document.querySelector("#OK"),
```

Comment:
Change to `btnConfirmOk` (Same for cancel). Nice job lining up the equals signs. :)